### ISSUE TITLE
drawImage(canvas, 0, 0)

### ISSUE THREAD

**Issue by 2xAA** [eendeego/node-openvg-canvas#30]
Official Canvas supports using the Canvas as an Image using context.drawImage.
Is there a possibility to do this with this library?

I'm not specifically asking for it to be added, just the theory behind it with either this library or OpenVG itself. I'd be more than happy working on implementing it myself.

From the [MozDocs](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/drawImage):
image
An element to draw into the context. The specification permits any canvas image source (CanvasImageSource), such as an HTMLImageElement, an HTMLVideoElement, an **HTMLCanvasElement** or an ImageBitmap.

I know this isn't an exact implementation, but there's any possibility to emulate this, it would be awesome.

Best,
Sam

**Comment by eendeego** [comment]
Yes, it can be done. OpenVG has an api to copy images from the framebuffer to a "memory" backed image.

Patches are welcome!

**Comment by 2xAA** [comment]
Well, I need this for a project I'm working on at University, so I'll get going on it and submit a pull request soon - thanks.
